app/services/trainees/create_from_apply.rb

```diff
@@ -43,7 +43,8 @@ def mapped_attributes
         email: raw_contact_details["email"],
         training_route: course&.route,
         subject: course&.name,
-        age_range: course&.age_range,
+        course_min_age: course&.min_age,
```

Comment:
Is there a reason we're preferring to set the `course_min_age` and `course_max_age` in this service, but setting `course_age_range: course&.age_range` in the `ConfirmPublishCourseForm`?

Reply:
No reason - good spot. I've made them both consistent.